src/main/scala/no/ndla/imageapi/controller/ImageControllerV2.scala

```diff
@@ -87,14 +85,14 @@ trait ImageControllerV2 {
         responseMessages(response404, response500))
 
     val newImage =
-      (apiOperation[ImageMetaInformation]("newImage")
```

Comment:
Må si jeg ikke er spesielt fan av navnet `ImageMetaInformationSingleLanguage`. Stemmer for å rename den til enten `ImageMetaInformationV2`, eller bare `ImageMetaInformation`. Hvis vi skal rename den til `ImageMetaInformation` bør også alle andre klasser som har en `V2`-suffix renames